pkg/collection/ducks.go

```diff
@@ -60,8 +60,9 @@ type DuckFilters struct {
 
 // NewDuckHunter
 // versions are used to default all the DuckType versions that apply to unfiltered CRDs.
-func NewDuckHunter(versions []v1alpha1.DuckVersion, filters *DuckFilters) DuckHunter {
+func NewDuckHunter(mapper ResourceMapper, versions []v1alpha1.DuckVersion, filters *DuckFilters) DuckHunter {
```

Comment:
I don't see this being used in the hunter yet. 
Planning on doing it in a follow up PR? Sounded weird all the UTs with the nil argument

Reply:
It is not used yet, next PR will use it.

Reply:
ack